src/foam/nanos/crunch/lite/CapablePayloadStatusDAO.js

```diff
@@ -60,9 +60,33 @@ foam.CLASS({
         if ( oldStatus == APPROVED && newStatus == PENDING ) {
           newStatus = APPROVED;
         }
+
+        if ( oldStatus == REJECTED  ) {
+          // TODO: if rejected, need to change the status of all children (prereqs) as well to rejected
```

Comment:
is this true? What if a prerequisite is common to another capability that is granted?

Reply:
Yeah so lol this isn't a todo anymore forgot to remove it. It exists for line 66 onwards. This is only preqreqs on the capable object though right so it will only happen for that object. Regardless if any of the higher levels get rejected on the invoice then everything really is rejected.